_data/universities/asu.yml

```diff
@@ -0,0 +1,5 @@
+name: "[Arizona State University](https://president.asu.edu/statements/statement-from-asu-president-michael-m-crow-on-the-transition-from-in-person-classes-to)"
```

Comment:
Would you mind renaming this file to `arizona-state.yml`? There are lots of ASUs out there (Alabama State, Arkansas State, etc.), and I'd rather not start a fight over which one gets to be the real ASU.

(This will get more important when considering Ohio State vs Oklahoma State vs Oregon State)